.github/workflows/trigger-builds.yml

```diff
@@ -3,7 +3,7 @@ name: Trigger builds for images and binaries
 on:
   workflow_dispatch:
   push:
-    branches: 
+    branches:
```

Comment:
@mwtian do we really need this change in this PR?

Reply:
Not strictly necessary, but just cleaning up since I browsed through file. I can revert if you prefer.

Reply:
We can leave it for sure, I am just always weary of formatting changes on top of code changes, since they tend to create un-needed churn.

Reply:
Sounds good, reverted!